helpers/installcredprovider.ps1

```diff
@@ -22,7 +22,12 @@ if ([Net.ServicePointManager]::SecurityProtocol.ToString().Split(',').Trim() -no
     [Net.ServicePointManager]::SecurityProtocol += [Net.SecurityProtocolType]::Tls12
 }
 
-$profilePath = [System.Environment]::GetFolderPath([System.Environment+SpecialFolder]::UserProfile)
+if ([System.Environment]::GetFolderPath([System.Environment+SpecialFolder]::UserProfile) -ne '') {
```

Comment:
Can you assign the return value of GetFolderPath to a variable and then check for it being empty? That way you don't  have to call that function twice.

Reply:
Thanks for your review, I've added the changes requested.